positron/modules/atom_common_v8_util.js

```diff
@@ -2,28 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-"use strict";
+'use strict';
```

Comment:
> Nit: other files use double quotes?

There's a mix of styles in this codebase. On this branch, the 20 JS files in positron/modules/ are evenly split between `"use strict";` and `'use strict';`. In general, I'm preferring single quotes these days, and converting files to use them, because it's the emerging standard in the Node community (and these files are Node modules, and Positron will eventually be published in NPM as a Node package).

See for example these recommendations from the top three Google search results for "node style guide":

* https://github.com/felixge/node-style-guide#use-single-quotes
* https://github.com/airbnb/javascript#strings--quotes
* https://github.com/RisingStack/node-style-guide#strings

That last link now suggests another style guide that recommends the same thing: http://standardjs.com/#rules